nntrainer/utils/node_exporter.cpp

```diff
@@ -59,22 +78,15 @@ void Exporter::saveTflResult(
   tf_node->setInOut(*self);
   tf_node->setInputs(self->getObject()->getInputRef());
   tf_node->setOutputs(self->getObject()->getOutputRef());
+
+  saveTflWeights(tf_node.get(), self->getRunContext(), "0:2:1");
```

Comment:
This behavior is specific to fc layer, could you check if this should move to line 89.

Reply:
right, this will be different for conv, bn, etc. Done.

Reply:
Let's delay the discussion later until we have some use case, I think we can easily change the behavior later, if we have other requirements.

Reply:
does "0:2:1" works for convolution layer?

Reply:
I think it is likely to be specific to fc. They transpose and save weights for fc.
confer below
https://github.com/tensorflow/community/blob/master/rfcs/20190722-tflite-training/trans1.png

Reply:
then layer needs to specify while declaring the weights if its weights are to be transposed for save/load or we have to maintain a list of layers whose weights are to be transposed.